notebooks/407-pedestrian-tracking-webcam/407-pedestrain-tracking.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>does we support changing reid models? We have several models in OMZ for this case? If yes, feature map size may be vary depends from model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/751/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes, since OMZ's person reidentification models are in same shape (1, 256). It should support all of them. If the length of model output changed, e.g (1, 128), it also works for cosine distance calculation.